Undeclared HWCAP_CRC32 compilation error in ARMv8 (Jetson TX2)

I'm getting this error:
`/home/nvidia/julia/src/crc32c.c:336:17: error: 'HWCAP_CRC32' undeclared (first use in this function)`
when compiling Julia master, at a0180d98a07ceec431f4af81db1000f6076610c8, in a [Jetson TX2 (ARMv8)](https://github.com/JuliaLang/julia/blob/master/README.arm.md#nvidia-jetson-tx2).
Two months older versions used to compile flawlessly.

Ok, guess that what I had compiled two months ago was v0.6.2, which still compiles since it source code does not uses `HWCAP_CRC32`.

Doing
```c
#include "processor.h"
#define HWCAP_CRC32 (1 << JL_AArch64_crc)
```
works.
But now I'm bumping into another, unrelated, error: 
```
ERROR: Invalid CPU name armv8-a.
Makefile:195: recipe for target '/home/nvidia/julia/usr/lib/julia/basecompiler.ji' failed
```

--- a/src/crc32c.c
+++ b/src/crc32c.c
@@ -43,6 +43,7 @@
 
 #include "julia.h"
 #include "julia_internal.h"
+#include "processor.h"
 
 #ifdef _CPU_AARCH64_
 #  include <sys/auxv.h>
@@ -333,7 +334,7 @@ JL_DLLEXPORT uint32_t jl_crc32c(uint32_t crc, const char *buf, size_t len)
 #  else
 static crc32c_func_t crc32c_dispatch(unsigned long hwcap)
 {
-    if (hwcap & HWCAP_CRC32)
+    if (hwcap & (1 << JL_AArch64_crc))
         return crc32c_armv8;
     return jl_crc32c_sw;
 }

